documentation/contributing/future_projects/index.markdown

```diff
@@ -8,9 +8,12 @@ title: Future Projects
 ---
 
 # Potential Code Sprints and Future Projects <!-- omit in toc -->
-
+<br />
+<p align="center">
 <img src="/assets/logo/moveit_logo-black.png" width="300"/>
```

Comment:
i think we should just remove this logo, its not needed

Reply:
What if we reuse this image instead?

```suggestion
<img src="/assets/images/people_page/people-illustration.png">
```

Reply:
![image](https://user-images.githubusercontent.com/561060/110699869-8cb59300-81ac-11eb-9d05-bf8feec1e528.png)